### PR TITLE
review of Portuguese sentences

### PR DESCRIPTION
**Pull request by arademaker**
So far, the test set has been checked. Please leave this PR open until I finish the training set. This PR is related to #22

### CODE REVIEW

**Comment by arademaker** [comment]
Done; please, this PR is ready to be accepted.
